GooglePlayInstant/Editor/AssetBundleVerifierWindow.cs

```diff
@@ -83,22 +87,33 @@ private static void UpdateAssetBundleVerificationInfoWindow()
                 // all objects that were loaded from this bundle.
                 bundle.Unload(true);
             }
+
+            // Turn request to null to be prepared for next call
+            www = null;
         }
 
-        private static double ConvertBytesToMegabytes(ulong bytes)
+        private double ConvertBytesToMegabytes(ulong bytes)
         {
             return bytes / 1024f / 1024f;
         }
 
-        private void OnGUI()
+        //TODO: fix malformed url behavior
+        private void Update()
         {
-            
-            UpdateAssetBundleVerificationInfoWindow();
+            if ((www != null) && (www.isDone))
+            {
+                getAssetBundleInfoFromDownload();
+                Repaint();
```

Comment:
Is this necessary every time or just the first after it's finished?

Reply:
Only necessary after it's finished.